src/components/Quiz/QuizzesStats.tsx

```diff
@@ -65,41 +107,23 @@ const QuizzesStats: React.FC = () => {
     ethereumBasicsQuizzes.length + usingEthereumQuizzes.length
   const totalQuizzesPoints = getTotalQuizzesPoints()
 
-  const computedAverage = average.length > 0 ? mean(average) : 0
-  // Normalize the raw average to be between 0 and 1 by dividing by 100
-  const normalizedAverage = computedAverage / 100
-
   // Data from Matomo, manually updated
   const collectiveQuestionsAnswered = 100000
-  const collectiveAverageScore = 67.4 / 100 // converted to fraction for percentage format
-  const collectiveRetryRate = 15.6 / 100 // converted to fraction for percentage format
-
-  // Todo: make global util function
-  const numberFormatter = new Intl.NumberFormat(localeForNumbers, {
-    style: "decimal",
-    minimumSignificantDigits: 2,
-    maximumSignificantDigits: 3,
-  })
+  const collectiveAverageScore = 67.4
+  const collectiveRetryRate = 15.6
```

Comment:
Can we move these to constants?

Reply:
That make sense, yes.

Reply:
Done https://github.com/ethereum/ethereum-org-website/pull/10349/commits/16e03f378b9fa3c47e54f1579345d17e51539274